jax/numpy_eval_lax.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2018 Google LLC
```

Comment:
As mentioned in another comment, I'd like to delete this file and only have lax_reference.py.